netlify.toml

```diff
@@ -0,0 +1,25 @@
+# Sentry uses netlify to deploy pull-request previews of the frontend which
+# talk to the production API.
+#
+# This is an EXPERIMENTAL developer experience, there are still various pages
+# that are served through django and have no SPA equivalent views.
```

Comment:
Do we have concerns about this being used maliciously by folks outside of the sentry org to publish things like password capture exploits and performing targetted phishing attacks?

Reply:
Anyone could setup a 'fake sentry' like this and perform targeted phishing attacks. So I don't think this is much of a concern. I do think there might still be a concern around the fact that these build previews could end up being more "trusted" looking.

I think what we can do is use the snippet injection feature of netlify to add some warning banners.

![image](https://user-images.githubusercontent.com/1421724/61086754-8e7fef80-a3e8-11e9-80b7-6cb02e540206.png)